sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
+
+\subsection{Researchers and staff affiliated with Flemish university associations}
+
+\begin{itemize}
+  \item Includes externally funded researchers registered in the personnel database (FWO, SBO, VIB, IMEC, etc.)
+  \item Includes researchers from all VSC partners
+  \item Usage is free of charge
+  \item Use your account credentials at your affiliated university to request a VSC-id and connect
+\end{itemize}
+
+
+\subsection{Researchers and staff affiliated with other Flemish or federal research institutes}
+
+\begin{itemize}
+  \item Includes researchers from e.g. INBO, ILVO, RBINS, etc.
+  \item Usage subject to an agreement with Ghent University
+  \item Every institute gets 1500 euro worth of compute time free of charge per year (corresponds to e.g. 90.000 core hours on cluster golett)
```

Comment:
In English, use `$$1,500$$` and `$$90,000$$` (the `$$` is just to get proper formatting).

Also, `golett` no longer exists...

Reply:
Point taken, removing the golett section

Reply:
Done